app/views/conversations/show.html.erb

```diff
@@ -59,7 +60,14 @@
   <div id="right-column" class="flex flex-1 flex-col h-screen bg-white dark:bg-graphite relative">
 
     <div id="handle-container" class="absolute left-0 top-1/2 w-8 h-20 -mt-10 z-20 hidden md:flex" data-click-toggle-target="trigger">
-      <div id="handle" class="flex-1 flex cursor-pointer group" data-controller="click-toggle" data-click-toggle-target="trigger" data-click-toggle-flippable-class="!hidden">
+      <div  id="handle"
+            class="flex-1 flex cursor-pointer group"
+            data-controller="click-toggle"
+            data-click-toggle-target="trigger"
+            data-click-toggle-flippable-class="!hidden"
+            data-action="keydown.meta+shift+s@document->click-toggle#handleEvent:prevent
+                         keydown.meta+period@document->click-toggle#handleEvent:prevent"
+            >
```

Comment:
@robacarp I realized that when the keydown event was only bound to the outer click-toggle then the sidebar opens & closes with the keyboard shortcut but the visual of the handle doesn't change properly. So I added a data-action here as well.

It kind of worked, but two issues:

1) I don't know why `:prevent` is on here but I saw you did it on the outer one and I figured you may have a good reason. I tried it without :prevent and it seemed to work fine so I'm inclined to remove it, but do we need it?

2) When I added it here then the binding events work properly when I click to focus on the document, however when the focus is inside the textarea (which is the default on load) then one of these two bindings fires twice! It's so odd. I load the page and press `Cmd+.` and I see 3 fires: 2x for one of them and 1x for another. I spent awhile trying to track it down and could not. It may have something to do with my nested controllers that I don't quite understand. In the end I solved it by simply adding a debouncer, which feels a little dirty but "good enough" for now.

Reply:
I know stimulus is all about generic controllers, but it sounds to me like it would be simpler to just have a sidebar controller and give it a few dedicated targets and actions. It feels like forcing the pattern to do this much work. 

```javascript
import { Controller } from "stimulus"

export default class extends Controller {
  static targets = [ "sidebar", "handle" ]

  toggle() {
    this.sidebarTarget.classList.toggle("md:flex")
    this.handleTargets.forEach( e => e.classList.toggle("hidden") )
  }
}
```